packages/react-ui-kit/src/Form/Checkbox.tsx

```diff
@@ -34,8 +34,10 @@ export interface StyledLabelProps<T = HTMLLabelElement> extends React.HTMLProps<
 
 const filterStyledLabelProps = (props: StyledLabelProps) => filterProps(props, ['markInvalid']);
 
-const checkSvg =
-  '<svg width="15" height="13" viewBox="0 0 16 13" xmlns="http://www.w3.org/2000/svg"><path d="M5.65685 12.0711L15.9842 1.62738L14.57 0.213167L5.65685 9.24264L1.41421 5L0 6.41421L5.65685 12.0711Z" fill="white"/></svg>';
+const checkSvg = (theme: Theme) =>
+  `<svg width="15" height="13" viewBox="0 0 16 13" xmlns="http://www.w3.org/2000/svg"><path d="M5.65685 12.0711L15.9842 1.62738L14.57 0.213167L5.65685 9.24264L1.41421 5L0 6.41421L5.65685 12.0711Z" fill="${
+    theme.darkMode ? 'black' : 'white'
+  }"/></svg>`;
```

Comment:
```suggestion
  `<svg width="15" height="13" viewBox="0 0 16 13" xmlns="http://www.w3.org/2000/svg"><path d="M5.65685 12.0711L15.9842 1.62738L14.57 0.213167L5.65685 9.24264L1.41421 5L0 6.41421L5.65685 12.0711Z" style={fille: ${theme.maincolor /* or equivalent */}}"/></svg>`;
```

Can we try this?

Reply:
If I'm not missing something, that's precisely what's not working :upside_down_face: